cmd/nerdctl/build.go

```diff
@@ -262,6 +262,10 @@ func generateBuildctlArgs(cmd *cobra.Command, buildkitHost string, platform, arg
 			}
 			needsLoading = true
 		}
+	} else {
```

Comment:
looks like we need `needsLoading=true` for `type=oci` too right ? https://github.com/containerd/nerdctl/pull/1146/files#diff-05bfdfb8af1ef5eb710a573250948e9c6935773725ddd08d162309b367e592bfR261

Reply:
Right. thanks for catching it.